### PR TITLE
DEVOP-1: Replace bridge-redis container with redis container

### PR DESCRIPTION
**Pull request by kare**
* Remove `bridge-redis`-container
* Use `redis`-container instead of `bridge-redis`-container

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -324,7 +324,7 @@ services:
     bridge_redis:
         command: [redis-server, --appendonly, 'yes']
         hostname: redis
```
Comment (kare) [inline docker-compose.yml:326]: @isentropy If the hostname is redis then does it mean that bridge_redis has been unused? Do you know whats the hostname for Streamr redis instance?

Comment (kare) [reply docker-compose.yml]: @isentropy hostname is now `bridge-redis` in master. I suppose that the Streamr redis hostname is `redis`.

**Comment by isentropy** [comment]
@kare Have you tested this with DU2 to verify that the bridge works? Should I?

**Comment by kare** [comment]
> @kare Have you tested this with DU2 to verify that the bridge works? Should I?

@isentropy Yeah, it works for me. Could you also test it?

**Pull request merged**